public/js/mongoTestClient.js

```diff
@@ -0,0 +1,89 @@
+const changeResponseText = (response) => {
+	const text = document.getElementById("responseText")
+	text.innerText = response.msg;
+	text.classList.add(response.status);
+	if (response.status === "success") {
+		text.classList.remove("fail")
+	} else {
+		text.classList.remove("success");
+	}
+}
+
+const setupEventListeners = () => {
+	document.getElementById("createButton").addEventListener("click", e => {
+		e.preventDefault();
+		fetch('/create-person', {
+			method: "POST",
+			headers: {
+				'Content-Type' : 'application/json'
+			},
+			body: JSON.stringify({
```

Comment:
I guess whether we send the request data in the body or the query parameters doesn't really matter, but I thought we were going to use query parameters. 😅 

If you don't want to change it, just say so.

Reply:
Oh, actually, don't change this! It makes sense for this to be in a body parameter, at least from reading [this blog post](https://wahlnetwork.com/2017/09/25/working-with-restful-api-query-body-and-path-parameters/).

General rule seems to be:
- Path parameters for getting a specific thing out of the API
    - For example: **GET** `/people/katy_p`
- Query parameters for filtering, or otherwise limiting the output of a query for lots of things
    - For example: **GET** `/people/katy_p/friends?limit=10&nameLike=Alex`
- Body parameters for supplying data needed to fulfill a request to the endpoint
    - For example: **POST** `/create-cat`, body: `{name:"Fluffy"}`